### PR TITLE
Implements #507. Appends region name to search bar placeholder text.

### PR DESCRIPTION
**Pull request by yogipatel**
Also removed the unused OBASearchResultsMapFilterToolbar class.

### CODE REVIEW

File: ui/search/OBASearchResultsMapViewController.m [hunk 1]
```diff
@@ -276,11 +272,11 @@ - (void)viewWillAppear:(BOOL)animated {
     [lm addDelegate:self];
     [lm startUpdatingLocation];
 
-    [self refreshSearchToolbar];
-
     if (self.searchController.unfilteredSearch) {
         [self refreshStopsInRegion];
     }
+    
+    self.searchBar.placeholder = [@"Search " stringByAppendingString:[OBAApplication sharedApplication].modelDao.region.regionName];
```
Comment (aaronbrethorst) [inline ui/search/OBASearchResultsMapViewController.m]: if ([OBAApplication sharedApplication].modelDao.region.regionName) {
      self.searchBar.placeholder = [NSString stringWithFormat:NSLocalizedString(@"Search %@",@""), [OBAApplication sharedApplication].modelDao.region.regionName];  
    }
    else {
      self.searchBar.placeholder = NSLocalizedString(@"Search", @"");
    }

for robustness, please.

**Comment by aaronbrethorst** [comment]
Looks :+1:, modulo the one change i mention. If you can change that and squash, I'll get it merged forthwith.

File: ui/search/OBASearchResultsMapViewController.m (hunk 1 above)
Comment (yogipatel) [inline ui/search/OBASearchResultsMapViewController.m]: 👍🏽 being away has made me lazy 🙃

**Comment by aaronbrethorst** [comment]
awesome thanks!

**Pull request merged**